run-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
For me the correct form should be:
`#!/usr/bin/env bash`
https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang

Reply:
That's a good point. I changed it because the `cookiecutter` made me do this. We should change it in the cookiecutter template as well then. I made as issue in the cookiecutter (https://github.com/inveniosoftware/cookiecutter-invenio-module/issues/114)